counter_contract_interaction_script/project/SPECIFICATION.md

```diff
@@ -0,0 +1,75 @@
+Table of Contents
+- [Overview of the Contract](#Overview-of-Contract)
+  - [Core Functionality](#core-functionality)
+    - [`increment()`](#increment)
+    - [`clear()`](#clear)
+  - [State Checks](#state-checks)
+    - [`count()`](#count)
+- [Overview of the Script](#Overview-of-Script)
+  - [Parameters](#Parameters)
+    - [`counter_contract_id`](#counter_contract_id:-ContractId)
+    - [`clear_count`](#clear_count:-bool)
+  - [Execution flow](#Execution-flow)
+- [Overview of the Rust integration test](#Overview-of-the-Rust-integration-test)
+
+# Overview of the Contract
+
+The contract has an internal variable `count` that can be retrieved or manipulated by the user based on the methods called.
+
+## Core Functionality
+
+This sub-section details what a user is able to do e.g. click a button and "x, y, z" happens.
+
+### `increment()`
+
+Increases the internal count variable by 1.
```

Comment:
```suggestion
- Increases the internal count variable by 1.
```

Reply:
Why just for this function? Or do you want that for all the fns?

Reply:
Taking a look at this again and comparing to other apps and when they use bullet points I'd say ignore my suggestion since this is a 1 line summary